collector/netdev_freebsd.go

```diff
@@ -21,11 +21,11 @@ import (
 import "C"
 
 const (
-	netDevSubsystem = "network"
+	subsystem  = "network"
```

Comment:
Similar comment as in your other PR about naming and package scope applies here.